script/staging/test/DeployEthereum.t.sol

```diff
@@ -417,59 +474,150 @@ contract DeployEthereumTest is Test {
         cctpBridge.lastDestinationLogIndex = cctpBridge.lastSourceLogIndex;
```

Comment:
This is only necessary if the messenger address is the same which is not the case on Base/Ethereum.